server/src/main/java/io/druid/curator/CuratorModule.java

```diff
@@ -67,18 +71,21 @@ public void configure(Binder binder)
   @Provides
   @LazySingleton
   public CuratorFramework makeCurator(
-      CuratorConfig config, EnsembleProvider ensembleProvider, Lifecycle lifecycle
-  )
+      CuratorConfig config, EnsembleProvider ensembleProvider, Lifecycle lifecycle)
   {
-    final CuratorFramework framework =
-        CuratorFrameworkFactory.builder()
-                               .ensembleProvider(ensembleProvider)
-                               .sessionTimeoutMs(config.getZkSessionTimeoutMs())
-                               .retryPolicy(new BoundedExponentialBackoffRetry(
-                                   BASE_SLEEP_TIME_MS, MAX_SLEEP_TIME_MS, MAX_RETRIES))
-                               .compressionProvider(new PotentiallyGzippedCompressionProvider(config.getEnableCompression()))
-                               .aclProvider(config.getEnableAcl() ? new SecuredACLProvider() : new DefaultACLProvider())
-                               .build();
+    final Builder builder = CuratorFrameworkFactory.builder();
+    if (null != config.getZkUser() && null != config.getZkPwd()) {
+      builder.authorization(config.getAuthScheme(),
+          (config.getZkUser() + ":" + config.getZkPwd()).getBytes(StandardCharsets.UTF_8));
```

Comment:
am wondering is there any specs about how to go from string to bytes?

Reply:
@b-slim - There isn't a spec., provided by ZK for charsets. Essentially, all fetches from ZK Server by the clients are base64 encoded. From the client-side using UTF-8 is more consistent across platforms than relying on platform's charset.
BTW., I tested it fine against ZK cluster running on AWS.